Sources/SwifterSwift/Foundation/NSAttributedStringExtensions.swift

```diff
@@ -180,4 +180,22 @@ public extension NSAttributedString {
     }
 }
 
+public extension Array where Element: NSAttributedString {
+    /// SwifterSwift: Returns a new `NSAttributedString` by concatenating the elements of the sequence, adding the given separator between each element.
+    ///
+    /// - Parameter separator: An `NSAttributedString` to add between the elements of the sequence.
+    /// - Returns: NSAttributedString with applied attributes.
+    // https://stackoverflow.com/questions/32830519/is-there-joinwithseparator-for-attributed-strings
+    func joined(separator: NSAttributedString) -> NSAttributedString {
+        guard let firstElt = first else { return NSMutableAttributedString() }
```

Comment:
Avoid using shorthands like "elt" for element. It can make code difficult to read.
```suggestion
        guard let firstElement = first else { return NSMutableAttributedString() }
```